rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+wrap_match_arms = true
+struct_lit_multiline_style = "ForceMulti"
+wrap_comments = false
+normalise_comments = false
```

Comment:
Perhaps it's best to use the standard and default style? That'd generally get the most bang-for-its-buck, right?